core/classes/AuthFlags.class.php

```diff
@@ -125,6 +136,13 @@ class AuthFlags {
 	 */
 	private $reauthentication_expiry = null;
 
+	/**
+	 * Indicates the autoprovisioning capability of the plugin
+	 * @var bool|false
```

Comment:
Remove `|false`, this is used in case of null because null is considered a different type than string for example, and hence you will see `string|null`.  But false is a valid bool value.

Reply:
Changed.